plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -44,6 +44,7 @@
 import com.thoughtworks.xstream.io.HierarchicalStreamReader;
 import com.thoughtworks.xstream.io.HierarchicalStreamWriter;
 import com.thoughtworks.xstream.mapper.Mapper;
+import groovyjarjarasm.asm.MethodTooLargeException;
```

Comment:
https://github.com/jenkinsci/workflow-cps-plugin/pull/817#discussion_r1408120262 still same issue

Reply:
PR updated in this direction